app/services/idv/steps/in_person/ssn_step.rb

```diff
@@ -24,7 +23,11 @@ def call
 
           idv_session.delete('applicant')
 
-          maybe_redirect_to_verify_info
+          # add this in its own pr for now
+          # if (IdentityConfig.store.in_person_verify_info_controller_enabled)
```

Comment:
Can remove the commented lines.

Reply:
resolved with commit 8103f5fab2b38b11803c84b605c17623f353b141